bamboo/bootstrap-unit-tests.sh

```diff
@@ -28,7 +28,15 @@ while ! docker container inspect ${container_id}\_build_env_1; do
 done
 
 ## Setup the build env container once it's started
-$docker_command "npm install --error --no-progress -g nyc; cd $UNIT_TEST_BUILD_DIR; git fetch --all; git checkout $GIT_SHA; npm install --error --no-progress; npm run bootstrap-no-build-quiet || true; npm run bootstrap-no-build-quiet"
+$docker_command "npm install --error --no-progress -g nyc; cd $UNIT_TEST_BUILD_DIR; git fetch --all; git checkout $GIT_SHA"
```

Comment:
Why are we removing the bootstrap here?  

bootstrap runs :

install
symlink dependencies
prepublish
prepare

If this is removed and code is added that adds a dependency, won't this not have the symlinked dependencies where they need to be?   I think it will  work, courtesy of the caching but it's not actually doing what we want, all of the time.       You also won't get the <new dependencies> behavior we currently have, as it will use whatever the cache generation was.

Reply:
I think this should be checkable via setting $USE_CACHED_BOOTSTRAP to false, so perhaps I'm missing something.

Reply:
The bootstrapping wasn't removed, it was just moved to line 39 so that it runs after the extraction of the TS build artifact, since the `prepare` step of bootstrapping relies on the existence of the compiled TS files throughout our packages. I did also remove the double bootstrapping here, but I can re-add.

I'm not following why we'd want to check for USE_CACHED_BOOTSTRAP=false though